lib/tasks/publishing_api.rake

```diff
@@ -0,0 +1,9 @@
+namespace :publishing_api do
+  desc "Send pages to the content-api"
+  task :register => :environment do
```

Comment:
So existing Capistrano task expects a `publishing_api:publish` rake task, unlike the panopticon and rummager ones which use `register`.  Would be good to update this so that we can use the shared deploy tasks.